Code/Selenium_Advanced/SeleniumAdvancedEpamTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+using System.Collections.ObjectModel;
+
+namespace Selenium_Advanced
+{
+    public class SeleniumAdvancedEpamTests
+    {
+        private IWebDriver _driver { get; set; }
+        private const string _epamUrl = "https://www.epam.com/";
+        private Actions action;
+        private WebDriverWait _wait;
+
+        [SetUp, Order(1)]
+        public void BrowserSetup()
+        {
+            _driver = new ChromeDriver();
+            _driver.Manage().Window.Maximize();
+            _driver.Navigate().GoToUrl(_epamUrl);
```

Comment:
navigation is not really a browser setup :)
you can create a thirs - test setup, where you will open main page each time before test and close cookie banner for example :)

Reply:
I have created  "AppMainPageSetUp()" for navigation on the main page.